__tests__/app.test.js

```diff
@@ -141,6 +141,33 @@ describe("TEST /api/articles", () => {
         });
       });
   });
+  test("Should accept a topic query and respond with the appropriate filtered articles", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: "mitch",
```

Comment:
Good assertion that makes sure we get the correct articles releveant to the topic 👍